packit/api.py

```diff
@@ -429,6 +429,12 @@ def update_dist_git(
             self._handle_sources(
                 force_new_sources=force_new_sources,
                 pkg_tool=pkg_tool,
+                env=self.common_env(version=version),
+            )
+        else:
```

Comment:
Why do you run it in the `else` branch? :eyes:

Reply:
So it runs even if upload to lookaside is disabled.

Reply:
I just don't see why it's mutually exclusive with `add_new_sources or force_new_sources`

Reply:
ah OK then :+1: